components/chat/chat-content/index.tsx

```diff
@@ -132,6 +132,8 @@ function ChatContent({ children, content, isChartChat, onLinkClick }: PropsWithC
     [context, cachePlguinContext],
   );
 
```

Comment:
what is the empty dom for

Reply:
Compatible summary format not user input